modules/reference/pages/feature/socialLogin/examples.adoc

```diff
@@ -7,13 +7,14 @@ The following examples show how to configure different scenarios in the `server.
 
 If multiple social media providers are configured, a customizable selection form is presented.
 The user can then sign in with a social media provider of their choice.
-The following example shows how to configure your application to request that the user logs in with their Google ID:
+The following example shows how to configure your application to request that the user logs in with their Google ID.
+In the case of users logging in with their Google ID, the Google client ID is "your_app_id" and the Client Secret is "your_app_secret".
```

Comment:
recommended edit:
In this example, `your_app_id` and  `your_app_secret` are the Google client ID and client secret credentials for the application that is defined in the `application` element.

In general, references to code elements and values in text descriptions should be in `monospace`, not quotes.

Reply:
Updated as : In this example, `your_app_id` and `your_app_secret` are the Google client ID and client secret credentials for the application that are defined in the `application` element.

Reply:
Updated as : In this example, `your_app_id` and `your_app_secret` are the Google client ID and client secret credentials for the application that are defined in the `application` element.